src/configs/prisma.ts

```diff
@@ -87,59 +87,61 @@ export const resetDatabase = (databaseUrl?: string) => {
 const prismaDemoComponent = `<script lang="ts" setup>
 const { data: examples } = useFetch('/api/examples')
 </script>
+
 <template>
   ${generateModuleHTMLComponent(
-    "Prisma ORM",
-    "Prisma unlocks a new level of developer experience when working with databases thanks to its intuitive data model, automated migrations, type-safety & auto-completion.",
-    "https://sidebase.io/sidebase/components/prisma",
+    'Prisma ORM',
+    'Prisma unlocks a new level of developer experience when working with databases thanks to its intuitive data model, automated migrations, type-safety & auto-completion.',
+    'https://sidebase.io/sidebase/components/prisma',
     `<p>
         Prisma ORM Data from the database, received {{ examples?.length || 0 }} records!
       </p>`,
-    "",
+    '',
   ).html}
 </template>
 `
 
 const prisma: ModuleConfig = {
-  humanReadableName: "Prisma ORM",
-  description: "Next-generation Node.js and TypeScript ORM. See more: https://www.prisma.io/",
+  humanReadableName: 'Prisma ORM',
+  description: 'Next-generation Node.js and TypeScript ORM. See more: https://www.prisma.io/',
+  scripts: [],
   dependencies: [
     {
-      name: "prisma",
-      version: "^5.7.1",
+      name: 'prisma',
+      version: '^5.11.0',
       isDev: true
     },
     {
-      name: "@prisma/client",
-      version: "^5.7.1",
+      name: '@prisma/client',
+      version: '^5.11.0',
       isDev: false
     }
   ],
   nuxtConfig: {},
   files: [{
-    path: ".env",
+    path: '.env',
```

Comment:
This may cause file overwrites. I don't know what the action would be here though

Reply:
Not an issue, as this CLI will only run if no folder exists in that location. Meaning that another `.env` cannot exist when this is created / ran!